components/nimbus/src/enrollment.rs

```diff
@@ -1445,6 +1445,88 @@ mod tests {
         .unwrap()
     }
 
+    fn get_conflicting_multi_feature_experiments() -> Vec<Experiment> {
+        serde_json::from_value(json!(
+            [
+            // This Experiment has three distinct features that a user can
+            // be involved in `non-conflicting-feature`, `conflicting-feature` and
+            // `some-non-conflicting-treatment`, any other experiment involving one of those
+            // would conflict with this experiment.
+            {
+            "schemaVersion": "1.0.0",
+            "slug": "another-monkey",
+            "endDate": null,
+            "branches":[
+                {"slug": "control", "ratio": 1, "features": [
+                    { "featureId": "non-conflicting-feature", "enabled": true },
+                    { "featureId": "conflicting-feature", "enabled": false}
+                ]},
+                {"slug": "treatment","ratio": 1, "features": [
+                    { "featureId": "some-non-conflicting-treatment", "enabled": true },
+                    { "featureId": "some-non-conflicting-treatment", "enabled": false },
+                    ]
+                },
+            ],
+            "channel": "nightly",
+            "probeSets":[],
+            "startDate":null,
+            "appName":"fenix",
+            "appId":"org.mozilla.fenix",
+            "bucketConfig":{
+                "count":1_000,
+                "start":0,
+                "total":10_000,
+                "namespace":"secure-silver",
+                "randomizationUnit":"nimbus_id"
+            },
+            "userFacingName":"2nd test experiment",
+            "referenceBranch":"control",
+            "isEnrollmentPaused":false,
+            "proposedEnrollment":7,
+            "userFacingDescription":"2nd test experiment.",
+            "id":"secure-silver",
+            "last_modified":1_602_197_222_372i64
+        },
+        // This experiment includes the feature `conflicting-feature` and
+        // will conflict with the above experiment
+        {
+            "schemaVersion": "1.0.0",
+            "slug": "yet-another-monkey",
+            "endDate": null,
+            "branches":[
+                {"slug": "control", "ratio": 1, "features": [
+                    { "featureId": "some-other-non-conflicting-feature", "enabled": true },
+                    { "featureId": "conflicting-feature", "enabled": false}
+                ]},
+                {"slug": "treatment","ratio": 1, "features": [
+                    { "featureId": "some_other_treatment", "enabled": true },
+                    { "featureId": "some_other_treatment", "enabled": false },
+                    ]
+                },
```

Comment:
Let's match the formatting in the branch above this and combine these to `]},` please